web/pages/api/auth/signin/index.js

```diff
@@ -38,4 +39,12 @@ const handler = nextConnect()
           }
     }) 
 
+passport.serializeUser(function(user, done) {
```

Comment:
passport.serializeUser and passport.deserializeUser should be configured in server.js and assigned before passport.initialize(). These are the callback functions that passport will use to figure out who is who.